laghos.cpp

```diff
@@ -296,7 +296,7 @@ int main(int argc, char *argv[])
     args.AddOption(&blast_energyFactor, "-bef", "--blastefactor", "Factor for scaling blast energy.");
     args.AddOption(&rom_paramID, "-rpar", "--romparam", "ROM offline parameter index.");
```

Comment:
It looks like the local variable `rom_paramID` can be removed and replaced by `romOptions.parameterID` which should then have the default value of -1 set in laghos_rom.hpp:73.